src/main/kotlin/no/skatteetaten/aurora/boober/utils/jsonNodeUtils.kt

```diff
@@ -173,6 +173,26 @@ fun JsonNode?.durationString(): Exception? {
     return null
 }
 
+fun JsonNode?.boolean(required: Boolean = true): Exception? {
+    val candidates = listOf("true", "false")
+
+    if (this == null) {
+        return if (required) {
+            IllegalArgumentException("Not a valid boolean value.")
```

Comment:
Kan vi endre lyden på exception for å differensiere den fra den under? "Missing required boolean value." f.eks.